parquet-column/src/main/java/org/apache/parquet/schema/GroupType.java

```diff
@@ -67,6 +67,16 @@ public GroupType(Repetition repetition, String name, OriginalType originalType,
     this(repetition, name, originalType, Arrays.asList(fields));
   }
 
+  /**
+   * @param repetition OPTIONAL, REPEATED, REQUIRED
+   * @param name the name of the field
+   * @param logicalTypeAnnotation (optional) the logical type to help with cross schema conversion (LIST, MAP, ...)
+   * @param fields the contained fields
+   */
+  public GroupType(Repetition repetition, String name, LogicalTypeAnnotation logicalTypeAnnotation, Type... fields) {
```

Comment:
I don't think we would like to expose this constructor. The builders are to be used by the API user.

Reply:
Good point!

Reply:
Unfortunately it seems that classes in parquet-hive module use this new constructor, that's why the Travis build failed. Reverted the constructor public.